extensions/amp-story/1.0/page-advancement.js

```diff
@@ -507,6 +507,9 @@ class TimeBasedAdvancement extends AdvancementConfig {
     /** @private @const {number} */
     this.delayMs_ = delayMs;
 
+    /** @private {?number} */
+    this.remainingDelayMs_ = null;
+
     /** @private {?number} */
     this.startTimeMs_ = null;
```

Comment:
Sorry, I'm not actually sure I even understand why this is important (although I am pretty sure I wrote this).  Isn't `delayMs` sufficient?

Reply:
It's used to compute the advancement %age for the progress bar :))